src/pages/youthboard/surveys/index.tsx

```
@@ -0,0 +1,128 @@
+import NoDataFound from '@/components/common/NoDataFound';
+import Header from '@/components/Header';
+import BackHeader from '@/components/youthNet/BackHeader';
+import Surveys from '@/components/youthNet/Surveys';
+import { surveysData, volunteerData } from '@/components/youthNet/tempConfigs';
```

ESLint

[error] 'volunteerData' is defined but never used.

(no-unused-vars)